src/Files.App/UserControls/SideBar/SideBarView.xaml.cs

```diff
@@ -30,6 +30,7 @@ public sealed partial class SidebarView : UserControl, INotifyPropertyChanged
 		public SidebarView()
 		{
 			InitializeComponent();
+			UpdateDisplayMode();
```

Comment:
Not sure this makes sense. When the constructor gets called, the UI has not loaded yet so updating visual states does not have much effect. Once the controlled is loaded the UI will change but in that case, we can also call this in the Loaded event.

Reply:
It is possible that the issue occurs elsewhere, but this solution works, if I leave it in the Loader method, the issue reappears.